src/battle-animations.ts

```diff
@@ -2716,24 +2717,28 @@ class PokemonSprite extends Sprite {
 		}
 
 		for (let i in pokemon.volatiles) {
-			status += PokemonSprite.getEffectTag(i);
+			status += PokemonSprite.getEffectTag(i, pokemon);
 		}
 		for (let i in pokemon.turnstatuses) {
 			if (i === 'roost' && !pokemon.getTypeList().includes('Flying')) continue;
-			status += PokemonSprite.getEffectTag(i);
+			status += PokemonSprite.getEffectTag(i, pokemon);
 		}
 		for (let i in pokemon.movestatuses) {
-			status += PokemonSprite.getEffectTag(i);
+			status += PokemonSprite.getEffectTag(i, pokemon);
 		}
 		let statusbar = this.$statbar.find('.status');
 		statusbar.html(status);
 	}
 
-	private static getEffectTag(id: string) {
+	private static getEffectTag(id: string, pokemon: Pokemon) {
 		let effect = PokemonSprite.statusTable[id];
 		if (typeof effect === 'string') return effect;
 		if (effect === null) return PokemonSprite.statusTable[id] = '';
 		if (effect === undefined) effect = [`[[${id}]]`, 'neutral'];
+		if (id === 'furycutter') {
+			let bp = pokemon.volatiles['furycutter'][2];
+			effect[0] = `Fury Cutter BP: ${bp * pokemon.volatiles['furycutter'][1]}`;
+		}
 		return PokemonSprite.statusTable[id] = `<span class="${effect[1]}">${effect[0].replace(/ /g, '&nbsp;')}</span> `;
```

Comment:
Bug: the HP bar effect will only update once, showing the BP of Fury Cutter doubled only once. Upon further usage of Fury Cutter, the tooltip will update as normal, but the bar effect will remain stuck.